packages/type-definitions/src/index.ts

```diff
@@ -110,34 +111,30 @@ const defaultTypesBundle: OverrideVersionedType[] = [
   },
 ]
 
-// Current runtime version: 10730
-export const typesBundle: OverrideBundleType = {
-  chain: {
-    'KILT Spiritnet': {
-      runtime: {
-        ...didCalls,
-        ...parachainStakingCalls,
-      },
-      types: defaultTypesBundle,
-    },
-    'KILT Peregrine': {
-      runtime: {
-        ...didCalls,
-        ...parachainStakingCalls,
-      },
-      types: defaultTypesBundle,
-    },
-    'KILT Mashnet': {
-      runtime: {
-        ...didCalls,
-      },
-      types: defaultTypesBundle,
-    },
-    Development: {
+// The KILT chains
+const chainNames: string[] = [
+  'KILT Spiritnet',
+  'KILT Spiritnet Develop',
+  'KILT Peregrine',
+  'KILT Peregrine Develop',
+  'KILT Mashnet',
+  'Development',
+]
+const chainConfiguration = chainNames.reduce(
+  (acc: Record<string, OverrideBundleDefinition>, name: string) => {
+    acc[name] = {
       runtime: {
         ...didCalls,
+        ...stakingCalls,
```

Comment:
It does not really, since they are not decorated if the metadata does not expose them. It is more a matter of clarity for whoever reads these type definitions. To be honest, since chain names do not change that often, I would keep it the way it was before and simply add the missing chains, as it could be that they will start diverging in the future (e.g., enabling public credentials only on one runtime).

Reply:
I agree. Will revert that.